src/main/java/seedu/address/model/medicalnote/MedicalNote.java

```diff
@@ -0,0 +1,102 @@
+package seedu.address.model.medicalnote;
+
+import static seedu.address.commons.util.AppUtil.checkArgument;
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Objects;
+
+/**
+ * Represents a medical note entry for a Patient,
+ * on a particular date by a particular doctor.
```

Comment:
Perhaps it is better to have the first sentence of a javadoc on a single line.

Reply:
Good eye 👍